bigtable-dataflow-import/src/test/java/com/google/cloud/bigtable/dataflowimport/ImportIntegrationTest.java

```diff
@@ -0,0 +1,265 @@
+import com.google.common.base.Function;
+import com.google.common.collect.Iterables;
+import com.google.common.collect.Sets;
+
+import org.apache.hadoop.hbase.Cell;
+import org.apache.hadoop.hbase.KeyValue;
+import org.apache.hadoop.hbase.util.Bytes;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.io.File;
+import java.util.Set;
+import java.util.UUID;
+
+/**
+ * Integration tests for importing HBase Sequence Files into Cloud Bigtable.
+ *
+ * <p>Three test cases are included:
+ * <ul>
+ *   <li>Sequence File in latest format on local file system.
+ *   <li>Sequence File in latest format on Google Cloud Storage (GCS).
+ *   <li>Sequence File in HBase 0.94 format on GCS.
+ * </ul>
+ *
+ * <p>To run this test, the following system properties must be defined:
+ * <ul>
+ *   <li>google.bigtable.project.id
+ *   <li>google.bigtable.cluster.name
+ *   <li>google.bigtable.zone.name
+ *   <li>google.dataflow.stagingLocation
+ *   <li>google.cloud.auth.service.account.email (for on-cloud tests)
+ *   <li>cloud.test.data.folder (on-cloud test data location)
+ * </ul>
+ *
+ * <p>Because {@link TestPipeline} is used for both local and on-cloud runs, tests
+ * in this classes must run sequentially (in arbitrary order). This is because {@code TestPipeline}
+ * use the {@code "runIntegrationTestOnService"} system property to choose the type of
+ * pipeline to create.
+ *
+ * <p>It is possible to use regular pipelines in these tests, e.g.,
+ * {@link com.google.cloud.dataflow.sdk.runners.DirectPipeline} or
+ * {@link BlockingDataflowPipelineRunner}, but
+ * <a href="https://cloud.google.com/dataflow/pipelines/testing-your-pipeline">Dataflow Testing
+ * Guide</a> recommends {@code TestPipeline}.
+ */
+@RunWith(JUnit4.class)
+public class ImportIntegrationTest {
+
+  // Location of test data hosted on Google Cloud Storage, for on-cloud dataflow tests.
+  private static final String CLOUD_TEST_DATA_FOLDER = "cloud.test.data.folder";
+  private static final String HBASE_094_SUBFOLDER = "hbase094";
+  private static final String HBASE_CURRENT_SUBFOLDER = "post-hbase094";
+
+  // Column family name used in all test bigtables.
+  private static final byte[] CF = Bytes.toBytes("column_family");
+
+  // TestPipeline configurations.
+  private static final String RUN_INTEGRATION_TEST_ON_SERVICE = "runIntegrationTestOnService";
+  private static final String TEST_PIPELINE_OPTIONS = "dataflowOptions";
+
+  private HBaseImportOptions commonOptions;
+  private String cloudTestDataFolder;
+  private String svcAccountEmail;
+
+  @Rule
+  public TemporaryFolder tmpFolder = new TemporaryFolder();
+
+  @Before
+  public void setup() {
+    commonOptions = PipelineOptionsFactory.as(HBaseImportOptions.class);
+    String projectId =
+        checkNotNull(System.getProperty(BigtableOptionsFactory.PROJECT_ID_KEY), "Project Id");
```

Comment:
nit: "Project ID" is not a error message. Something like "Project ID is required" ?